build.py

```diff
@@ -99,7 +99,7 @@ def GetVisualStudioCompilerAndVersion():
         # VisualStudioVersion environment variable should be set by the
         # Visual Studio Command Prompt.
         match = re.search(
-            "(\d+).(\d+)",
+            r"(\d+)\.(\d+)",
```

Comment:
When using python 3 this gives the following warning:
````
build.py:102: DeprecationWarning: invalid escape sequence \d
      "(\d+).(\d+)",
````
I believe you got this function from the USD build script. I checked and they made this fix a while ago. The env var has something like "15.0" in it. The **r** tells python this is a raw string which should not be interpreted. And you need **\.** to find the . otherwise that is the regex matching any char.

Reply:
Ugh I see. Great find. Didn't realize that.